README.md

```diff
@@ -114,12 +114,26 @@ From Conda
 conda install pytorch-lightning -c conda-forge
 ```
 
-Install bleeding-edge (no guarantees)   
+#### Install bleeding-edge - future 1.2
+
+the actual status of 1.2 [nightly] is following:
+
+![CI base testing](https://github.com/PyTorchLightning/pytorch-lightning/workflows/CI%20base%20testing/badge.svg?branch=release%2F1.2-dev&event=push)
+![CI complete testing](https://github.com/PyTorchLightning/pytorch-lightning/workflows/CI%20complete%20testing/badge.svg?branch=release%2F1.2-dev&event=push)
+![PyTorch & Conda](https://github.com/PyTorchLightning/pytorch-lightning/workflows/PyTorch%20&%20Conda/badge.svg?branch=release%2F1.2-dev&event=push)
+![TPU tests](https://github.com/PyTorchLightning/pytorch-lightning/workflows/TPU%20tests/badge.svg?branch=release%2F1.2-dev&event=push)
+![Docs check](https://github.com/PyTorchLightning/pytorch-lightning/workflows/Docs%20check/badge.svg?branch=release%2F1.2-dev&event=push)
+
+Install future release from the source (no guarantees)   
+```bash
+pip install git+https://github.com/PytorchLightning/pytorch-lightning.git@release/1.2-dev --upgrade
```

Comment:
Choose between `-U` or `--upgrade`. Below you are using `-U`

Reply:
t for some users -U is not intuitive so I would rather use full name